bridges/N26Bridge.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class N26Bridge extends BridgeAbstract
+{
+	const MAINTAINER = 'quentinus95';
+	const NAME = 'N26 Blog';
+	const URI = 'https://n26.com';
+	const CACHE_TIMEOUT = 1800;
+	const DESCRIPTION = 'Returns recent blog posts from N26.';
+
+	public function getIcon()
+	{
+		return 'https://n26.com/favicon.ico';
+	}
+
+	public function collectData()
+	{
+		$html = getSimpleHTMLDOM(self::URI . '/en-fr/blog-archive')
+			or returnServerError('Error while downloading the website content');
+
+		foreach($html->find('div.ga') as $article) {
+			$item = [];
+
+			$item['uri'] = self::URI . $article->find('h2 a', 0)->href;
+			$item['title'] = $article->find('h2 a', 0)->plaintext;
+
+			$fullArticle = getSimpleHTMLDOM($item['uri']);
+
+			$item['timestamp'] = strtotime($fullArticle->find('.fk.fl.de.ch.fm.by'));
```

Comment:
This line throws lots of error messages for multiple reasons:

* `find` returns an array if no index is specified.
* Chaining classes doesn't work right now. They can be matched exactly though.
* The return value of `find` is an object, which doesn't work with `strtotime`. The plaintext of that object seems to work for me.

```suggestion
			$dateElement = $fullArticle->find('span[class="fk fl de ch fm by"]', 0);
			$item['timestamp'] = strtotime($dateElement->plaintext);
```